pkg/clustermesh/common/clustermesh.go

```diff
@@ -44,6 +44,14 @@ type Configuration struct {
 
 	// Metrics holds the different clustermesh metrics.
 	Metrics Metrics
+
+	// FeatureMetrics will track which features are enabled with in clustermesh.
+	FeatureMetrics ClusterMeshMetrics
```

Comment:
This variable is currently never initialized, neither in production code nor in tests.